.gitignore

```diff
@@ -2,4 +2,9 @@ build/
 dist/
 Example_Python_Package.egg-info/
 .idea/
-venv/
\ No newline at end of file
+venv/
+COAsT/__pycache__/
```

Comment:
We should probably consider putting a .gitignore in each directory, rather than one master file for every directory